src/Reducer/filterReducer.js

```diff
@@ -0,0 +1,37 @@
+export const filterReducerFtn = (state,action) => {
+    switch(action.type) {
+        case "SORT_BY_PRICE" :
+            return {...state , sortBy : action.payload}
+        case "PRICE_RANGE" :
+            return {...state , priceRange : action.payload}
+        case "RATE_BY_STAR" :{
+            console.log(state.ratingBy)
```

Comment:
You can remove console.log if not required

Reply:
done